fcrepo-http-api/src/main/java/org/fcrepo/http/api/responses/StreamingBaseHtmlProvider.java

```diff
@@ -164,10 +164,6 @@ void init() throws IOException {
 
         templatesMap = templatesMapBuilder
             .put(REPOSITORY_NAMESPACE + "RepositoryRoot", velocity.getTemplate(getTemplateLocation("root")))
-            .put(REPOSITORY_NAMESPACE + "Binary", velocity.getTemplate(getTemplateLocation("binary")))
```

Comment:
I think we should probably keep these for rendering these types (except, 'Pairtree').
Is there a reason why you believe they should go away?

Reply:
Seemingly a `fedora:Binary` will always also have a `ldp:NonRDFSource` type and therefore will get the template using that type. Same with `fedora:Container` and `ldp:RDFSource`. So having these seemed redundant.